learn/wavemaker-release-notes/v10-11-1.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "WaveMaker 10.11.0 - Release date: 24 January 2022"
+id: ""
+sidebar_label: "Release v10.11.1"
+---
+*WaveMaker released another version update with new features and bug fixes.*
+
+---
+
+## Bug Fixes
+
+Upgraded Log4j2 dependency version to mitigate against the vulnerability [CVE-2021-44832](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44832).
+
+- Log4j2 upgraded to **2.17.1**.
+
+:::note
+To learn more about Log4j2 dependency workaround, see [Mitigating Log4J 2 Vulnerability](/learn/blog/2021/12/13/zero-day-vulnerability).
+:::
+
+
+### Data Table
+
+- Fixed an issue related to the multicolumn filter in Data Table for sorting dates that are not aligned correctly for Default Theme.
+- Fixed an issue related to setting the width of the columns in the advance settings of a Data Table; the table in the inactive tab does not load with the column width as defined.
```

Comment:
Fixed an issue related to the width of the columns of table widget which is kept inside tabs widget. When the page is loaded and the table is rendered inside an inactive tab, the width assigned to the columns was not getting reflected properly when the tab was selected. This is now fixed.